spec/models/work_statistic_spec.rb

```diff
@@ -374,10 +365,10 @@
             ws.recalculate
             expect(ws.complete).to eq(0)
 
-            create(:page, work: work, status: Page::STATUS_TRANSCRIBED)
-            create(:page, work: work, status: Page::STATUS_BLANK)
-            create(:page, work: work, status: Page::STATUS_INDEXED)
-            create(:page, work: work, status: Page::STATUS_NEEDS_REVIEW)
+            create(:page, work: work, status: :transcribed)
```

Comment:
Okay, so here's a place where we have access to `:transcribed` from a spec file, so don't have to use a string literal.  Is it because we're using it within `status:`?  _I'm so confused..._

Reply:
Let me check this again. We should be able to just use `create(:page, work: work, status: :whatever_status)`, but I think I did use string literals for the other cases because of FactoryBot. But I will review and apply changes